desktop/assets/texts/ro-RO/preferences_dialog.ftl

```diff
@@ -0,0 +1,37 @@
+preferences-dialog = Preferințe Ruffle
+preference-locked-by-cli = Numai Citire (Setat de CLI)
+graphics-backend = Grafică Backend
+graphics-backend-default = Prestabilit
+graphics-power = Preferință de Alimentare
+graphics-power-low = Minim (e.g. iGPU)
+graphics-power-high = Maxim (e.g. GPU)
+language = Limbaj
+audio-output-device = Dispozitiv de Ieșire Audio
+audio-output-device-default = Prestabilit de sistem
+enable-openh264 = Activează OpenH264
+show-license = Afişează Licenţa
+openh264-license = Licență OpenH264
+log-filename-pattern = Nume Fișier Jurnal
+log-filename-pattern-single-file = Fișier Unic (ruffle.log)
+log-filename-pattern-with-timestamp = Cu Marcaj Temporal
+storage-backend = Locația fișierelor de salvare
+storage-backend-disk = Disc
+storage-backend-memory = Memorie
+recent-limit = Limita Recentelor
+recent-clear = Golire
+theme = Temă
+theme-system = Implicit
+theme-light = Lumină
+theme-dark = Întuneric
+# See for context https://github.com/FeralInteractive/gamemode
+gamemode = Modul Joc
+gamemode-tooltip =
+    Modul Joc aplică temporar un set de optimizări computerului și/sau aplicației Ruffle.
+    Ruffle solicită Modul Joc atunci când se redă un film.
+gamemode-default = Prestabilit
+gamemode-default-tooltip = Modul Joc va fi activat numai când preferința de alimentare este setată la maxim.
```

Comment:
Commented on CrowdIn about GameMode being translated, and substituted it in place of "Modul Joc".